Source/Core/DolphinWX/ConfigMain.cpp

```diff
@@ -577,6 +580,7 @@ void CConfigMain::CreateGUIControls()
 	ConfirmStop = new wxCheckBox(DisplayPage, ID_INTERFACE_CONFIRMSTOP, _("Confirm on Stop"));
 	UsePanicHandlers = new wxCheckBox(DisplayPage, ID_INTERFACE_USEPANICHANDLERS, _("Use Panic Handlers"));
 	OnScreenDisplayMessages = new wxCheckBox(DisplayPage, ID_INTERFACE_ONSCREENDISPLAYMESSAGES, _("On-Screen Display Messages"));
+	PauseOnFocusLost = new wxCheckBox(DisplayPage, ID_INTERFACE_PAUSEONFOCUSLOST, _("Pause on Focus Lost"));
```

Comment:
@JosJuice There is no explicit language rule on things like this (the link just takes one arbitrary example set of rules). What matters is to moderately make sense and, more importantly, to be consistent, and I think having "lost" (being an adjective) lowercase here is consistent with the rest of our GUI.